### PR TITLE
Update to use stellar-etl with BufferedStorageBackend

### PR DESCRIPTION
**Pull request by chowbao**
Update with changes from https://github.com/stellar/stellar-etl/pull/242

### CODE REVIEW

**Comment by chowbao** [comment]
Update stellar-etl image after [PR](https://github.com/stellar/stellar-etl/pull/242) merged

File: airflow_variables_dev.json [hunk 1]
```diff
@@ -279,5 +279,5 @@
   "elementary_secret": "slack-token-elementary",
   "dbt_elementary_dataset": "test_elementary",
   "use_captive_core": "False",
-  "txmeta_datastore_url": "gcs://exporter-test/ledgers/testnet"
+  "txmeta_datastore_path": "ledger-exporter/ledgers"
```
Comment (sydneynotthecity) [inline airflow_variables_dev.json:282]: does this mean that testnet and pubnet ledgers are saved in the same bucket?

**Review by sydneynotthecity — APPROVED** [review]
LGTM

File: airflow_variables_dev.json (hunk 1 above)
Comment (chowbao) [reply airflow_variables_dev.json:282]: Yes. They are both in the same bucket with a path `ledger-exporter/ledgers/pubnet` and `ledger-exporter/ledgers/testnet`

It determines `testnet` or `pubnet` based on the new `env.Network` field

**Comment by chowbao** [comment]
Wait for https://github.com/stellar/stellar-etl-airflow/pull/356 to be merged first

**Pull request merged**